src/main/java/com/elharo/propernouns/Names.java

```diff
@@ -60,8 +103,13 @@ private static boolean isNameWord(String word) {
       }
     }
     
+    // Check if the word (case-insensitive) is in our known names set
```

Comment:
word = word.toLowerCase(Locale.Root)

and then don;t need to lowercase it or create a separate variable below